### ISSUE TITLE
Please release the latest PHP8 compatibility changes

### ISSUE THREAD

**Issue by Gummibeer** [jfvoliveira/blade-boxicons#3]
Hey,

right now the PHP8 changes aren't released - so as a PHP8 user I have to require `dev-master` (not cool) or can't use the package. 😟

**Comment by alperyazgan** [comment]
Waiting for Laravel8.x support (blade-ui-kit/blade-icons 1.0.x) ...

**Comment by Gummibeer** [comment]
Check out the clone by @mallardduck it contains all changes and introduced a v2 because of vendor/namespace changes and so on. But for you as a user nothing really changed - so you should be able to change the composer.json and it just works.
https://packagist.org/packages/mallardduck/blade-boxicons

**Comment by mallardduck** [comment]
Also - I've just updated my package with a update generator...i'd be lying if I wasn't inspired by the generator that @Gummibeer and I have worked on for spatie's emoji package...but anyway, that's a thing that I've added which has allowed me to sync with boxicons upstream.

TL;DR: my package at version 2.1.0 now has all icons that box icons published to NPM as of today.

**Comment by alperyazgan** [comment]
Yesterday I download the repo of this package and and on a private copy I patched some that will correct the issue and then I saw the reply from @Gummibeer and decided to go with the real one, now I am using your package @mallardduck, thanks a lot indeed.
